test/basic.js

```diff
@@ -1,76 +1,73 @@
-'use strict';
+import assert from "power-assert";
+import * as ShioriJK from "shiorijk";
+import {ShioriTransaction} from "../src/lib/shiori_transaction";
 
-if (typeof require !== 'undefined') {
-  var assert = require('power-assert');
-  var ShioriJK = require('shiorijk');
-  const shiori_transaction = require('../src/lib/shiori_transaction');
-  var ShioriTransaction = shiori_transaction.ShioriTransaction;
-}
-
-
-describe('ShioriTransaction', () => {
-  let request3, request2, response3, response2;
+describe("ShioriTransaction", () => {
+  let request3;
+  let request2;
+  let response3;
+  let response2;
   beforeEach(() => {
     request3 = new ShioriJK.Message.Request({
       request_line: {
-        method: 'GET',
-        version: '3.0',
+        method:  "GET",
+        version: "3.0",
       },
       headers: {
-        ID: 'OnBoot',
-        Charset: 'UTF-8',
-        Sender: 'Ikagaka',
+        ID:      "OnBoot",
+        Charset: "UTF-8",
+        Sender:  "Ikagaka",
       },
     });
 
     request2 = new ShioriJK.Message.Request({
       request_line: {
-        method: 'GET Sentence',
-        version: '2.6',
+        method:  "GET Sentence",
+        version: "2.6",
       },
       headers: {
-        Event: 'OnBoot',
-        Charset: 'UTF-8',
-        Sender: 'Ikagaka',
+        Event:   "OnBoot",
+        Charset: "UTF-8",
+        Sender:  "Ikagaka",
       },
     });
 
     response2 = new ShioriJK.Message.Response({
       status_line: {
-        code: 200,
-        version: '2.6',
+        code:    200,
+        version: "2.6",
       },
       headers: {
-        Sentence: '\\h\\s[0]\\e',
-        Charset: 'UTF-8',
-        Sender: 'ikaga',
+        Sentence: "\\h\\s[0]\\e",
+        Charset:  "UTF-8",
+        Sender:   "ikaga",
       },
     });
 
     response3 = new ShioriJK.Message.Response({
       status_line: {
-        code: 200,
-        version: '3.0',
+        code:    200,
+        version: "3.0",
       },
       headers: {
-        Value: '\\h\\s[0]\\e',
-        Charset: 'UTF-8',
-        Sender: 'ikaga',
+        Value:   "\\h\\s[0]\\e",
+        Charset: "UTF-8",
+        Sender:  "ikaga",
       },
     });
   });
 
-  it('can initialize', () => {
+  it("can initialize", () => {
     assert(new ShioriTransaction());
   });
 
-  it('to() will convert 3 <-> 2', () => {
+  it("to() will convert 3 <-> 2", () => {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->